openedx/core/lib/block_structure/manager.py

```diff
@@ -110,4 +111,5 @@ def clear(self):
         Removes cached data for the block structure associated with the given
         root block key.
         """
+        print "clear() in manager.py ***************"
```

Comment:
Ah.. just curious - wanted to make sure it wasn't because of a lack of log statements.  If needed for testing/validation, we can add additional ones.